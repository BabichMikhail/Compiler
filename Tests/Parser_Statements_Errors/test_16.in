begin 
	for i := 10 to 50 do begin
		if (a > b) then
			break
		else if a + b > 10 ;
			then
			continue
	end  
end.
