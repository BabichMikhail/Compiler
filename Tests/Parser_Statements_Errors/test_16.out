Fatal: Syntax error, "then" expected but ";" found
